src/components/Sep8Send/Sep8ActionRequiredForm.tsx

```diff
@@ -0,0 +1,128 @@
+import { useEffect, useState } from "react";
+import { useDispatch } from "react-redux";
+import { Button, Input, Loader } from "@stellar/design-system";
+import { Heading2 } from "components/Heading";
+import { Modal } from "components/Modal";
+import {
+  initiateSep8SendAction,
+  sep8SendActionRequiredFieldsAction,
+} from "ducks/sep8Send";
+import { useRedux } from "hooks/useRedux";
+import { ActionStatus, Sep8Step } from "types/types.d";
+
+export const Sep8ActionRequiredForm = ({
+  onClose,
+}: {
+  onClose: () => void;
+}) => {
+  const { account, sep8Send } = useRedux("account", "sep8Send");
+  const [fieldValues, setFieldValues] = useState<{ [key: string]: string }>({});
+  const {
+    actionFields,
+    message,
+    actionMethod,
+    actionUrl,
+  } = sep8Send.data.actionRequiredInfo;
+  const { nextUrl, result } = sep8Send.data.actionRequiredResult;
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (sep8Send.data.sep8Step === Sep8Step.SENT_ACTION_REQUIRED_FIELDS) {
+      if (result === "follow_next_url") {
```

Comment:
We could make this a const to be reused.

Reply:
Good point, added in https://github.com/stellar/stellar-demo-wallet/pull/164/commits/4c2da9929b65672f6660aa7a82062221e1c56f50.